---
title: 新型冠状病毒（COVID-19）传播情况的基因组学分析：状况报告 2020-05-15.
authors:
  - Sidney M. Bell
  - Emma Hodcroft
  - Nicola Müller
  - Cassia Wagner
  - James Hadfield
  - Richard Neher
  - Trevor Bedford
authorLinks:
  - https://twitter.com/sidneymbell
  - https://neherlab.org/emma-hodcroft.html
  - https://bedford.io/team/nicola-mueller/
  - https://bedford.io/team/cassia-wagner/
  - https://bedford.io/team/james-hadfield/
  - https://neherlab.org/richard-neher.html
  - https://bedford.io/team/trevor-bedford/
affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; CZI, CA, USA"
translators: 
  - Fengjun Zhang
  - 李雪莹
translatorLinks:
  - https://twitter.com/fengjun_zhang
  - https://xueyingcli.weebly.com/
license: "CC-BY"
licenseLink: "https://creativecommons.org/licenses/by/4.0/"
dataset: "https://nextstrain.org/ncov/global/2020-05-14?d=tree&l=clock&legend=closed"

abstract: "本报告使用公开共享的基因组学数据来追踪 SARS-CoV-2 的传播和演化。这周我们将总结病毒的碱基突变和这些突变给 COVID-19 大流行可能带来的（和不可能带来的）影响。"
---
<!-- Translators: Only text after : in the above ^ needs to be translated -->
<!-- Comment tags like these do not need to be translated, they are only to help you! -->
<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->

<!-- This is left-side text -->
# [报告摘要](https://nextstrain.org/ncov/2020-05-14?d=tree,entropy&p=grid)

我们分析了 5,193 个公开共享的 SARS-Cov-2 基因组。通过这些病毒基因组间的相互比对，我们可以了解 COVID-19 在世界和本地范围的传播过程。在本周的更新中，我们重点关注病毒的碱基变异，报告包括以下内容：
<br><br>
* 什么是碱基突变的来源（DNA碱基突变是一个自然过程）  
* SARS-CoV-2 的碱基突变速率 （也没有很惊人）
* 有多少 SARS-CoV-2 的病毒株正在传播（我们目前已知只有一株）  
* 地理文化和实际发生的疫情，对病毒的基因型带来怎样的显著影响（关于这一点非常难说）

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 2-->
# [COVID-19 背景资料](https://nextstrain.org/ncov/global/2020-05-14?d=tree&p=full&legend=closed)

#### Nextstrain 的资料
* [起点基础教程：如何解读系统发生树](https://nextstrain.org/narratives/trees-background/zh).
* [之前的疫情状况报告](https://nextstrain.org/ncov-sit-reps/).
* [详细解说（英文推特帖）](https://bedford.io/misc/twitter/).
* [冠状病毒的背景知识（英文）](https://nextstrain.org/help/coronavirus/human-CoV).
* [常见误区](https://nextstrain.org/narratives/ncov/sit-rep/zh/2020-03-13?n=11).

#### 外部资源
* [冠状病毒家族如何突变碱基，如何传播（纽约时报英文）](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).
* [向科学家提问 & 常见问题解答（英文）](https://covid19.fas.org/l/en).
* [WHO 的状况报告（英文）](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
* [美国 CDC 的资料（英文）](https://www.cdc.gov/coronavirus/2019-ncov/index.html).

<!-- This is the right-side text -->
```auspiceMainDisplayMarkdown

# 状况报告间章
<p>
在疫情早期，我们并不清楚 SARS-CoV-2 如何在国家之间传播，也不知它如何在地区内蔓延，更不知已呈现爆发的地区疫情来源。
之前的四个月，我们在状况报告里连续追踪了以下（及更多其他）问题，而现在的疫情阶段，我们已清楚病毒是如何在国家和地区之间，打破一些边境后蔓延的模式。
</p>
<p/>

- 即使是全世界相隔遥远的地区，疫情爆发也显著地存在相互影响。<p/>

- 通过人类的移动和旅行，病毒是被重复多次传入大部分国家。<p/>  

- 在这些小型的“火种”流入新社区之后，许多并未引起广泛传播便自己消失了。然而在某些地区特征和偶然可能性下，一些“火种”演变成为地区性的疫情爆发。<p/>  

- 最后，这些地区性的爆发形成了自己的“火种”，接着又向其他新的地域传播下去。  
<p/>

我们在第一轮经历感染潮的国家中发现这样的传播模式实属情有可原，但需要警惕的是，我们在一些离最初疫情爆发后几个月，再次传入新病例的国家中，也观察到这样的传播模式。
最终，只在有效地实施病毒检测，病例追踪和及时隔离的国家中，这样的模式才被打破。
<br><br>

这表示全世界的疫情爆发都紧密地联系在一起，这场与 COVID-19 抗争的战疫将会是全球性的，若不督促世界上的每一处角落都重视，我们最终会战胜不了它。
<br><br>

在疫情发展的这个阶段，结合地理的流行病学是各地域以及更细分的地域内，社区人群最紧迫需要从当地公共卫生机构知道的。
<br><br>

因此，这将是我们这阶段最后一份周报，不过我们依旧会提供蕾丝的报告给各地域的公共卫生机构。
<br><br>

我们当然会继续我们的每日更新，一旦有心绪发布，我们就会更新我们网站上全球和各地域的病毒信息。
如果之后的疫情中发生什么发规模的新事件，我们也会再撰写额外的状况报告。
所有的这些信息更新将会第一时间发布在 [Nextstrain 的英文官方推特账号](https://twitter.com/nextstrain) 。  
<br><br>

你也可以在 <a href="https://nextstrain.org/ncov-sit-reps/">这里</a> 找到我们所有的状况报告。

```

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [病毒基因组随时间而改变实属正常](https://nextstrain.org/ncov/global/2020-05-14?c=num_date&d=tree,entropy&m=div&p=full&legend=open)

导致 COVID-19 的致病病毒，新冠病毒SARS-CoV-2，是一个 [冠状病毒科](https://nextstrain.org/help/coronavirus/human-CoV) 的病毒。 新冠病毒和其他冠状病毒一样，是由RNA而非DNA编码的病毒。
<br><br>
病毒之所以会感染细胞，是因为它需要借用细胞中的工具来自我复制。RNA病毒的自我复制过程是非常容易出错的，因为大多数RNA聚合酶（能够复制RNA的分子）没有校对功能，不能自我纠错。这会导致病毒基因组中出现频繁的突变，这是正常现象，也是我们意料之中的。
<br><br>
重要的是，大部分的突变要么会“破坏”病毒，让它不能传播和/或复制，要么不会产生任何影响，由于[基因组编码的冗余性](https://en.wikipedia.org/wiki/Synonymous_substitution)。
也就是说，这样的突变在任何意义上都不会改变病毒的蛋白质乃至其功能。
还有一些突变可能会非常轻微地改变病毒的蛋白质，但不会对其功能产生任何影响。
在非常罕见的情况下，遗传密码中的改变会帮助病毒更好地复制和/或传播，但几乎所有这类改变都仍只能造成非常轻微的影响。

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Variation in viruses' genetic code is useful for tracing outbreaks](https://nextstrain.org/ncov/global/2020-05-14?c=gt-ORF3a_57&d=tree,entropy&f_division=New%20York&m=div&p=full)

These differences in the genomes between viruses can be used as "bread crumbs" to trace the history of a given sample.
Just as the people in your family share unique combinations of genetic characteristics, closely related viral samples share unique combinations of genetic mutations.
<br><br>
For example, here we show the viral "family tree" with each sample colored by which amino acid is present at a specific location in the viral genome (gene "ORF3a", site 57).
<br><br>
Highlighting samples from New York, we see that most of the cases from New York and Europe have the amino acid Histidine ("H") rather than Glutamine ("Q") at this position. This, in combination with all the other sites in the genome, tells us that these cases are all closely related to each other.
<br><br>
You can also see the position in gene "ORF3a" highlighted on the chart at the bottom of the left.
<br><br>
For a more in-depth explanation of these concepts, we recommend [this visual explanation](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html) by Jonathan Corum and Carl Zimmer.


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [The evolutionary rate of SARS-CoV-2 is typical for a coronavirus](https://nextstrain.org/ncov/global/2020-05-14?c=num_date&d=tree,entropy&l=clock&p=full)

Because viral genomes inevitably change, these differences tend to accumulate at a steady rate over time.
<br><br>
Here, the x axis shows the date that each sample was collected. The y axis shows the *total number* of mutations each strain is away from the root of the tree. Each sample is colored by the date it was collected.
<br><br>
While there are certainly some outliers, on average, we see changes accumulate at a rate of ~24 substitutions per year. This means that if a single viral lineage were transmitting from one person to the next for a full year, we'd expect its entire genome to accumulate roughly 24 substitutions by the end of the year.
As the whole SARS-CoV-2 genome is ~30,000 bases, this corresponds to roughly ~1 mutation per 1,000 bases in a year.
<br><br>
For context, influenza would average ~2 mutations per 1,000 bases per year; HIV would average ~4 mutations per 1,000 bases per year.

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [As far as we know, there is only 1 strain of SARS-CoV-2](https://nextstrain.org/ncov/global/2020-05-14?branchLabel=clade&c=clade_membership&d=tree&p=full)

There have been many reports about multiple "strains" of SARS-CoV-2.
First, a clarification: when virologists use the word "strain," it's often just as a useful label so we can all refer to the same group of samples (e.g., our clade labels, shown here). This does *not* imply any known functional difference.
<br><br>
Separately, "strain" can be used to refer to viral genotypes that are functionally distinct, either biologically (e.g., pathogenicity/disease severity) and/or epidemiologically (e.g., transmissibility).
Importantly, though, determining whether two genotypes actually *are* functionally distinct requires much more experimental, clinical and epidemiological data than we currently have.
<br><br>
One of the most prominent hypotheses about SARS-CoV-2 strains compares possible "D614" vs "G614" strains.


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [D614G may be related to transmissibility, but could also be explained by geography](https://nextstrain.org/ncov/global/2020-05-14?c=gt-S_614&gmax=25778&gmin=21082&p=full)

[A recent pre-print](https://www.biorxiv.org/content/10.1101/2020.04.29.069054v1) suggests that a single mutation, D614G (from amino acid "D" to "G" at site 614 in the "S" ('spike') protein), may be responsible for increasing the transmissibility of SARS-CoV-2.
<br><br>
In many geographic areas, the relative frequency of 614D compared to 614G increased over time.
If a certain genotype leads to better transmissibility, we would expect it's frequency to increase over time.
There are, however, other explanations to why the relative frequency of some genotypes would increase over time.
<br><br>
For this site, we see a number of geographic locations where initially, most viral lineages in circulation had the D allele; later, this balance flips such that most lineages have the G allele.
This *could* be a hallmark of G being slightly "fitter" than D (e.g., if viruses with the G mutation had a slightly higher R0 through increased transmissibility).
<br><br>
However, this could also just be a side effect of the natural history of the pandemic.
Early on in the pandemic, most strains exported from China had a D allele. Later, most strains from Italy had a G allele. Recently, we've seen more exportation from Europe than from Asia overall (although there's certainly some sampling bias at play here, as well).
Thus, it could just be that this particular genotype got lucky, and spread so rapidly because it got there first.
<br><br>
There's a fair amount of debate regarding the relative merits of these two hypotheses. It'll be important to carefully tease these apart, but we don't have scientific consensus on this issue yet. For a more detailed explanation, [see this thread](https://twitter.com/trvrb/status/1257825352660877313).


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 13-->
# [What you can do](https://nextstrain.org/ncov/2020-05-14?c=country&d=map&p=full)
#### ...as an individual
* Practice strict social distancing, especially if you are in a vulnerable group.
* Remember that even if you are not super vulnerable, many people around you are; follow these practices to protect others.
* Wash your hands "like you just chopped a jalapeno and have to change a contact lens."
* Stay home as much as possible -- especially  if you are sick; be prepared with extra supplies in case you need to self-quarantine.
* If you are an employer, encourage your employees to work from home wherever possible.

#### ...as an official
* Make testing free and broadly available.
* Put strong social distancing measures in place.
* Fund and implement extensive contact tracing efforts.
* Financially support those impacted by social distancing measures.


<!-- This is the right-side text -->
```auspiceMainDisplayMarkdown
# Takeaways  

#### Viruses naturally acquire mutations as they replicate. This is normal. Mutations can also help us trace the course of epidemics.  

#### SARS-CoV-2 mutation rate is very typical for coronaviruses.

#### As far as we know, there is only 1 functional "strain" of SARS-CoV-2.  

#### Teasing apart the biological impact of specific mutations is tricky -- many perceived differences can be attributed to chance and epidemiological factors.  
```

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text -->
# [Scientific credit](https://nextstrain.org/ncov/global/2020-05-14?d=tree&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.
<br><br>
**We encourage you to click on 'Explore the Data Yourself' and scroll down for a full list of authors; the author of each individual sequence is available by selecting it on the tree.**
<br><br>
We also gratefully acknowledge GISAID for providing the platform through which these data can be uploaded and shared.
